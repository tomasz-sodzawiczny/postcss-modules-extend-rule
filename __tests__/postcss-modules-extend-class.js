import extendClass from "../src/plugins/postcss-modules-extend-class";

const process = css => extendClass.process(css).css;

describe("postcss-modules-extend-class", () => {
  it("works", () => {
    const css = ".a { @postcss-modules-extend-class __some_class_name; }";

    expect(process(css)).toMatchSnapshot();
  });
});
