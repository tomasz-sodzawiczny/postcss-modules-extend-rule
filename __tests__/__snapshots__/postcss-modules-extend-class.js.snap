// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`postcss-modules-extend-class works 1`] = `".a { @extend .__some_class_name; }"`;
